### ISSUE TITLE
Got Unable to read master data in KanColle Panel

### ISSUE THREAD

**Issue by leeyc0** [KC3Kai/KC3Kai#2619]
Please make sure that you have these details on hand when submitting your issue/bug report as this will greatly speed up the issue triage process -- thank you!

- Your Chrome version - 68.0.3440.106 64 bit
- Your OS and its version Windows 10 1709
- 32bit or 64bit? 64 bit
- Your KC3 Kai version - 32.8.0
- Which build you have tried running? webstore version as in the one from [Chrome Web Store]
- A KC3 Error Report JSON file if you have faced an API error. To upload it, you might append `.txt` as file name extension.

I got this error after HTML5 upgrade (and transfer from Sukumo to Shortland):

```
Unable to read master data. Here are possible reasons why:
- You opened this F12 Panel too late. Please open the F12 devtools panel before Game Start button.
- Guys I'm not really sure how you've been opening the game. But please read the instructions on-screen.
- Clicking "Just Let Me Play Now" will opt you out of those features (panel info). The game intro pages say that you'll need to press F12 panel on those screens, not after you start the game, not anywhen else.
Again... Play via [method] -> F12 -> KanColle tab -> Game automatically starts

```
Resetting profile doesn't help. I got RemodelDB error as attached.
[[9161337] ErrorLog 2018-08-17.json.txt](https://github.com/KC3Kai/KC3Kai/files/2297371/9161337.ErrorLog.2018-08-17.json.txt)

**Comment by Colonel-Colt** [comment]
Same bug here.

**Comment by mikenzb** [comment]
Fixed in 33.0.0
